src/TimeStamp.cpp

```diff
@@ -1,47 +0,0 @@
-#include <iostream>
```

Comment:
Don't delete this file. This is a good example to demonstrate functionality without ros, and to test that the imu is working correctly (or what frame it's in, or do statistical analysis).

Reply:
This still lives in the FRC900 repo as a standalone. It serves no purpose for us here.

Reply:
It does. Trust me from experience, it is a nice way to verify that the hardware is working. Having a demo exe that can check/isolate issues with a hardware device is critical for saving time/energy. For example, we have cansend/dump exes for the canbus.